doc/source/building/index.rst

```diff
@@ -125,18 +125,18 @@ your system.
 
         brew install gfortran openblas pkg-config
 
-    .. note::
+    To allow the build tools to find OpenBLAS, you must run::
+
+        brew info openblas | grep PKG_CONFIG_PATH
 
-        ``export PKG_CONFIG_PATH="/opt/homebrew/opt/openblas/lib/pkgconfig"``
-        may need to be used in order for the build system to detect OpenBlas.
+    This will give you a command starting with ``export PKG_CONFIG_PATH=``, which
+    you must run.
 
     .. note::
 
-        As of SciPy >=1.2.0, we do not support compiling against the system
-        Accelerate library for BLAS and LAPACK. It does not support a sufficiently
-        recent LAPACK interface. This is planned to change in 2023, because macOS
-        13.3 introduced a major upgrade to Accelerate which resolved all known
-        issues.
+        As of SciPy 1.14.0, we have added support for the Accelerate library
+        for BLAS and LAPACK. It requires macOS 13.3 or greater. This support
+        can be enabled with ``python dev.py build --with-accelerate``.
```

Comment:
`dev.py` shouldn't be used at this point in the docs, since this section isn't specific to contributors rather than people who just want to build from source (i.e., end users and packagers).

I suggest changing the last sentence of this paragraph with a link to [the page about BLAS and LAPACK](http://scipy.github.io/devdocs/building/blas_lapack.html#selecting-blas-and-lapack-libraries). Something like "to build with Accelerate instead of OpenBLAS, see the instructions at (link)".

And then on that page, add a second example block under _"... include `mkl`, `atlas` and `blis`."_ (and also name `accelerate` there), giving both the `dev.py` and `python -m build` invocations.

Reply:
> `dev.py` shouldn't be used at this point in the docs, since this section isn't specific to contributors rather than people who just want to build from source (i.e., end users and packagers).
>
> I suggest changing the last sentence of this paragraph with a link to [the page about BLAS and LAPACK](http://scipy.github.io/devdocs/building/blas_lapack.html#selecting-blas-and-lapack-libraries). Something like "to build with Accelerate instead of OpenBLAS, see the instructions at (link)".

Got it. I changed that to a link in dc017e508a2f8deec0d5435cb59247738a783b78.

>(and also name `accelerate` there)

I added `accelerate` to the list of BLAS options.

I also tested whether those commands worked with Accelerate.

I tried `python -m build -Csetup-args=-Dblas=accelerate -Csetup-args=-Dlapack=accelerate`, and `python dev.py build -C-Dblas=accelerate -C-Dlapack=accelerate`, and both commands work.

> And then on that page, add a second example block under _"... include `mkl`, `atlas` and `blis`."_ (and also name `accelerate` there), giving both the `dev.py` and `python -m build` invocations.

I added this in 9ea71ddbd23283b14fe9fc364de84401e4885838, but I am not sure I fully understood what you meant.

I added two commands:

    $ python dev.py build --with-accelerate
    $ python -m build -Csetup-args=-Dblas=accelerate -Csetup-args=-Dlapack=accelerate

The first command seems fine - it's documenting something which you couldn't figure out from reading the previous examples.

The second command I added is just the previous code block with `accelerate` replacing `blas` everywhere. Is this necessary? It seems inconsistent to document how to do this, but only for one alternate BLAS backend. Maybe I'm missing something and there's some simpler way to do this. :)

Reply:
Good points @nickodell. I've added some more info, about `--with-scipy-openblas` and simplifying the Accelerate wheel build command. That also addresses your last point about whether it's necessary at all.